service/discovery/azure/azure_properties.go

```diff
@@ -260,3 +260,18 @@ func location(region *string) *ontology.GeoLocation {
 		Region: util.Deref(region),
 	}
 }
+
+// publicNetworkAccessStatus returns the public access status of the resource
+func publicNetworkAccessStatus(status *string) bool {
```

Comment:
I think we should stick with the old name because we had a lot of trouble with what exactly "public network access status" is